scripts/algolia/transform-mdx-to-algolia-record/create-records.mjs

```diff
@@ -55,13 +55,16 @@ export async function createAlgoliaRecordObject(
 	const codeListItems = await collectCodeListItems(markdownFile)
 
 	return {
-		objectID,
-		page_title: frontmatter.page_title,
-		type: 'docs',
-		products: [productSlug],
-		description: frontmatter.description ?? '',
-		headings,
-		codeListItems,
-		_tags: [directory],
+		action: 'addObject',
+		body: {
+			objectID,
+			page_title: frontmatter.page_title,
+			type: 'docs',
+			products: [productSlug],
+			description: frontmatter.description ?? '',
+			headings,
+			codeListItems,
+			_tags: [directory, 'web-unified-docs'],
```

Comment:
Added the tag `web-unified-docs` as a troubleshooting measure in case we need to verify which records in the index are coming from UDR